tests/unit/broccoli/ember-app-test.js

```diff
@@ -1088,9 +1088,9 @@ describe('EmberApp', function() {
 
   describe('vendorFiles', function() {
     let defaultVendorFiles = [
-      'jquery.js',
       'ember.js',
       'app-shims.js',
+      'jquery.js',
```

Comment:
Is this order important? I assume it's not.

Reply:
The order matters _somewhat_ because `jquery.js` _must_ end up in `dist/assets/vendor.js` **before** Ember itself...